cmd/mattermost/commands/extract_content.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/shared/mlog"
+)
+
+var ExtractContentCmd = &cobra.Command{
+	Use:     "extract-documents-content",
+	Short:   "Extracts the documents content",
+	Long:    "Extracts the documents content and stores it in the database for document searche",
+	Example: "extract-documents-content --from=12345",
+	RunE:    extractContentCmdF,
+}
+
+func init() {
+	ExtractContentCmd.Flags().Int64("from", 0, "The timestamp of the earliest file to extract, expressed in seconds since the unix epoch.")
+	ExtractContentCmd.Flags().Int64("to", model.GetMillis(), "The timestamp of the latest file to extract, expressed in seconds since the unix epoch.")
+	RootCmd.AddCommand(ExtractContentCmd)
```

Comment:
How about using `Duration` instead of `int64` for user input? It accepts all values that `time.ParseDuration` can parse so its a lot friendlier for the end user than specifying unix epochs.

Reply:
This is very specific one-time-thing that shouldn't be used frequently. And it is consistent with the current interface for other cases. The idea of add a "duration" instead of the `to` makes it a bit weird because you are mixing both, start time and duration instead of start time and end time.

This is consistent with the `export` command that use epoch milliseconds for define from where to export.